cmd/envtool/envtool.go

```diff
@@ -410,6 +417,65 @@ var cli struct {
 }
 
 func main() {
+
+	ctx := kong.Parse(&cli)
+
+	logging.Setup(zap.InfoLevel, "")
+
+	if cli.Debug {
+		logging.Setup(zap.DebugLevel, "")
+	}
+
+	logger := zap.S()
+
+	seedCorpus, err := os.Getwd()
+	if err != nil {
+		logger.Fatal(err)
+	}
+
+	generatedCorpus, err := generatedCorpus()
```

Comment:
That logic should be in a separate function similar to `shellRead`, `packageVersion`, etc, not just in the `main` function